MdeModulePkg/Core/Dxe/DxeMain.h

```diff
@@ -1575,6 +1578,19 @@ CoreWaitForEvent (
   OUT UINTN     *UserIndex
   );
 
+// MU_CHANGE begin
+
+/**
+  Initialize the memory protection special region reporting.
+**/
+VOID
+EFIAPI
+CoreInitializeMemoryProtectionSpecialRegions (
+  VOID
+  );
+
+// MU_CHANGE end
+

```

Comment:
```suggestion

 // MU_CHANGE End
```

Reply:
This is uncrustify enforced. It thinks it is weird (rightly so, to not have an empty line between the last comment and the next, but that means we can't have the MU_CHANGE be the last line added.